en/flight_modes/acro_mc.md

```diff
@@ -0,0 +1,41 @@
+# Acro Flight Mode (Multicopter)
+
+<!-- this requires review and updates -->
+
+[<img src="../../assets/site/difficulty_5.svg" title="Difficulty (Hard)" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;
+
+*Acro mode* is the RC mode for performing acrobatic maneuvers e.g. flips, rolls and loops.
+
+The roll, pitch and yaw sticks control the rate of angular rotation around the respective axes and throttle is passed directly to the output mixer. When sticks are centered the vehicle will stop rotating, but remain in its current orientation (on its side, inverted, or whatever) and moving according to its current momentum.
+
+![MC Manual Acrobatic Flight](../../images/flight_modes/manual_acrobatic_MC.png)
+
+<!-- image above incorrect: https://github.com/PX4/px4_user_guide/issues/182 -->
+
+## Technical Description
+
+RC/manual mode for performing acrobatic maneuvers e.g. flips, rolls and loops.
+
+RC RPY stick inputs control the rate of angular rotation around the respective axes. When sticks are centered the vehicle will stop rotating, but remain in its current orientation (not necessarily level!)
+
+## Parameters
+
+Parameter | Description
+--- | ---
+&nbsp; | 
+
+ 
+<!-- 

```

Comment:
Below are possibly relevant parameters. Should these be made visible? Are there any others?

Reply:
yes, all of these are important

Reply:
Thanks. Added